stable/redis/templates/svc.yaml

```diff
@@ -7,6 +7,11 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+{{- if .Values.metrics.enabled }}
```

Comment:
I'm not sure if we should always add these annotations when metrics are enabled. Maybe this should be configurable. You don't need these annotations if you use the Prometheus Operator.

@mgoodness What do you think?